microsoft-edge/progressive-web-apps-chromium/how-to/icon-theme-color.md

```diff
@@ -60,10 +60,219 @@ Each icon in the `icons` array should come with at least the `src` and `sizes` p
 
 Learn more about [the icons member](https://developer.mozilla.org/docs/Web/Manifest/icons).
 
-<!-- TODO: add information about which sizes Windows require at a minimum, and which sizes are used where -->
+## Icon Image Sizes
 
+Your PWA can be enhanced on Windows by supplying the images with specific dimensions in your web app manifest. We recommend choosing one of the options below:
+
+* Level 1: (Basic image support: 512x512) This is the base image from which to generate missing images
```

Comment:
Missing period at the end of this sentence.

Reply:
fixed